src/components/keyboardnavigation.js

```diff
@@ -60,11 +65,28 @@ define(["inputManager", "layoutManager"], function (inputManager, layoutManager)
         return KeyNames[event.keyCode] || event.key;
     }
 
+    /**
+     * Returns _true_ if key is used for navigation.
+     *
+     * @param {string} key name
+     * @return {boolean} _true_ if key is used for navigation
+     */
+    function isNavigationKey(key) {
+        return NavigationKeys.indexOf(key) != -1;
+    }
+
     function enable() {
         document.addEventListener("keydown", function (e) {
```

Comment:
Shouldn't we be adding the event listener only if in TV layout, instead of registering an event simply to perform the check and return ?

Reply:
At first I did it as you said - just return from `enable`. But then I thought about the possibility of using media buttons in other layouts, therefore cut off only those that interfere. I have no media buttons on my keyboard. Emulated through global hotkeys are not worked - separate buttons are captured. `Pause` works because everyone has it.

Do you suggest to completely disable the keyboard?

Reply:
Fair enough, I hadn't though about media buttons.
We can leave it as-is then.